### PR TITLE
Fixed inactivity timer

### PR DESCRIPTION
**Pull request by 409H**
## Description

The inactivity timer view will show the correct minutes of inactivity instead of static "5 minutes"

### CODE REVIEW

File: common/v2/features/ScreenLock/ScreenLockLocking.tsx [hunk 1]
```diff
@@ -86,7 +93,7 @@ export default class ScreenLockLocking extends Component<LockScreenProps> {
         <ContentWrapper>
           <Title>{translate('SCREEN_LOCK_LOCKING_HEADING')}</Title>
           <Description>
-            {translate('SCREEN_LOCK_LOCKING_DESCRIPTION')}{' '}
+            {translate(`SCREEN_LOCK_LOCKING_DESCRIPTION`, { $inactive_time: ((settings.inactivityTimer/1000)/60).toString() })}{' '}
```
Comment (vilsbole) [inline common/v2/features/ScreenLock/ScreenLockLocking.tsx]: use `moment.duration().format` or equivalent: https://stackoverflow.com/questions/13262621/how-do-i-use-format-on-a-moment-js-duration

**Comment by vilsbole** [comment]
Is this still WIP

**Comment by FrederikBolding** [comment]
Fixed the formatting issue, rebased and changed the component to be functional so we could use `SettingsContext`.

Tested and it works. Should be ready for review!

**Comment by FrederikBolding** [comment]
Btw, this one should be working. For some reason the AWS CodeBuild thing failed initially, but not when retrying.

**Pull request merged**